pkg/api/openshiftcluster.go

```diff
@@ -17,10 +19,29 @@ type OpenShiftCluster struct {
 	Name       string                     `json:"name,omitempty"`
 	Type       string                     `json:"type,omitempty"`
 	Location   string                     `json:"location,omitempty"`
+	SystemData *SystemData                `json:"systemData,omitempty"`
```

Comment:
after discussion, we think that SystemData should not be a pointer, but its individual fields all should be.  See https://stackoverflow.microsoft.com/questions/242534